tools/codegen/core/gen_config_vars.py

```diff
@@ -276,18 +237,12 @@ def string_default_value(x, name):
         fuzz = attr.get("fuzz", False)
         if not fuzz:
             continue
+        assert fuzz == True
```

Comment:
This assert isn't doing much because of the `if not fuzz:` 2 lines above. Are you worried about the type (e.g., `fuzz = "arst"`)?

Reply:
yeah... it used to take that syntax, now it doesn't: makes sense to assert that for a bit.

Reply:
Python isn't going to help a ton there, but alright.